.github/workflows/on_pr_master.yml

```
@@ -132,9 +132,9 @@ jobs:
         run: |
           if [ "$test_consul_acl_default_policy" == 'consul_acl_allow' ]
           then
-              echo "::set-output name=default_policy::allow"
+              echo "ENVKEY_consul_acl_default_policy=true" >> $GITHUB_ENV
           else
-              echo "::set-output name=default_policy::deny"
+              echo "ENVKEY_consul_acl_default_policy=false" >> $GITHUB_ENV
```

```suggestion
              echo "ENVKEY_consul_acl_default_policy=deny" >> $GITHUB_ENV
```
The tests seem to have failed because of this typo.

---

Changed :D